MailCore/Utils/Constants.swift

```diff
@@ -161,8 +161,16 @@ public enum Constants {
         let release = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as! String? ?? "x.x"
         let build = Bundle.main.object(forInfoDictionaryKey: "CFBundleVersion") as! String? ?? "x"
         let betaRelease = Bundle.main.isRunningInTestFlight ? "beta" : ""
+        let platformDetector = PlatformDetector()
+        var device = ""
```

Comment:
You can do it with a ternary operator. Do you see how to do it?
You can look at the variables above

The name device bothers me as it's the system name or os name